README.md

```diff
@@ -171,24 +176,24 @@ assert %{status: 403, message: "not authorized"} = error
 
 ## Schema Scopes
 
-Bodyguard also provides the `Bodyguard.Schema` behaviour to query which items a user can access. Implement it directly on schema modules.
+Bodyguard also provides the `Bodyguard.Schema` behaviour to filter out items a user shouldn't access. Implement it directly on schema modules.
 
 ```elixir
 defmodule MyApp.Blog.Post do
   import Ecto.Query, only: [from: 2]
   @behaviour Bodyguard.Schema
```

Comment:
It might be a good idea to make this `use Bodyguard.Schema` if you want to be able to eventually inject functionality.  Not a big deal, just something that might prevent a breaking change down the line.